src/RestQuery.js

```diff
@@ -188,7 +188,7 @@ function RestQuery(
 // Returns a promise for the response - an object with optional keys
 // 'results' and 'count'.
 // TODO: consolidate the replaceX functions
-RestQuery.prototype.execute = function(executeOptions) {
+RestQuery.prototype.execute = function (executeOptions) {
```

Comment:
Just a quick note here: All of these function parenthesis space additions are a result of the Prettier update as well. Prettier's most recent release includes a change to function parenthesis spacing that cannot be disabled. 

Full discussion here: https://github.com/prettier/prettier/pull/3903